Source/FilteringFunctions/arm_biquad_cascade_df2T_f64.c

```diff
@@ -29,415 +29,574 @@
+ The advantage of the Direct Form II Biquad is that it requires half the number of state variables, 2 rather than 4, per Biquad stage.
+ 
+ @par           Instance Structure
+ The coefficients and state variables for a filter are stored together in an instance data structure.
+ A separate instance structure must be defined for each filter.
+ Coefficient arrays may be shared among several instances while state variable arrays cannot be shared.
+ 
+ @par           Init Functions
+ There is also an associated initialization function.
+ The initialization function performs following operations:
+ - Sets the values of the internal structure fields.
+ - Zeros out the values in the state buffer.
+ To do this manually without calling the init function, assign the follow subfields of the instance structure:
+ numStages, pCoeffs, pState. Also set all of the values in pState to zero.
+ @par
+ Use of the initialization function is optional.
+ However, if the initialization function is used, then the instance structure cannot be placed into a const data section.
+ To place an instance structure into a const data section, the instance structure must be manually initialized.
+ Set the values in the state buffer to zeros before static initialization.
+ For example, to statically initialize the instance structure use
+ <pre>
+ arm_biquad_cascade_df2T_instance_f64 S1 = {numStages, pState, pCoeffs};
+ arm_biquad_cascade_df2T_instance_f32 S1 = {numStages, pState, pCoeffs};
+ </pre>
+ where <code>numStages</code> is the number of Biquad stages in the filter;
+ <code>pState</code> is the address of the state buffer.
+ <code>pCoeffs</code> is the address of the coefficient buffer;
+ */
 
 /**
-  @addtogroup BiquadCascadeDF2T
-  @{
+ @addtogroup BiquadCascadeDF2T
+ @{
  */
 
 /**
-  @brief         Processing function for the floating-point transposed direct form II Biquad cascade filter.
-  @param[in]     S         points to an instance of the filter data structure
-  @param[in]     pSrc      points to the block of input data
-  @param[out]    pDst      points to the block of output data
-  @param[in]     blockSize number of samples to process
-  @return        none
+ @brief         Processing function for the floating-point transposed direct form II Biquad cascade filter.
+ @param[in]     S         points to an instance of the filter data structure
+ @param[in]     pSrc      points to the block of input data
+ @param[out]    pDst      points to the block of output data
+ @param[in]     blockSize number of samples to process
+ @return        none
  */
-
-
+#if defined(ARM_MATH_NEON)
 void arm_biquad_cascade_df2T_f64(
-  const arm_biquad_cascade_df2T_instance_f64 * S,
-  const float64_t * pSrc,
-        float64_t * pDst,
-        uint32_t blockSize)
+	const arm_biquad_cascade_df2T_instance_f64 * S,
+	const float64_t * pSrc,
+	float64_t * pDst,
+	uint32_t  blockSize)
 {
+	
+	
+	
+	const float64_t *pIn = pSrc;                  /*  source pointer            */
```

Comment:
When I look at our dft2_f32, I conclude that this implementation can be optimized much more.
You should compute several stages per loop iteration.
(But it could come in a future PR. It won't be blocking for having this one accepted).